plugins/text-freetype2/text-functionality.c

```diff
@@ -141,10 +141,56 @@ void set_up_vertex_buffer(struct ft2_source *srcdata)
 	}
 
 skip_word_wrap:;
+
+	// calculate lines width for alignment
+	len = wcslen(srcdata->text);
+	uint32_t current_line_width = 0;
+	uint32_t current_line_index = 0;
+	if (srcdata->custom_width < 100) {
```

Comment:
Style:  Do not use curly braces for this `if-else` block.  But do leave a blank line after it where the last brace would have been.